### ISSUE TITLE
gRPC TracingServerInterceptor not added by the agent

### ISSUE THREAD

**Issue by fuleow** [open-telemetry/opentelemetry-java-instrumentation#1430]
I have two services running the OTEL agent 0.9.0 with opentelemetry-sdk 0.9.1 and the first one calls the second using gRPC. The `TracingClientInterceptor` from the caller appears to be applied correctly because on the second service's debug logs I see the `traceparent` in the header.

```
2020-10-20 09:16:34.540 DEBUG [grpc-nio-worker-ELG-3-2] io.grpc.netty.shaded.io.grpc.netty.NettyServerHandler : [id: 0xd32a7495, L:/127.0.0.1:9092 - R:/127.0.0.1:58619] INBOUND HEADERS: streamId=3 headers=GrpcHttp2RequestHeaders[:path: /com.my.Company/Method, :authority: localhost:9092, :method: POST, :scheme: http, te: trailers, content-type: application/grpc, user-agent: grpc-java-netty/1.27.2, traceparent: 00-a1842f26d466a2242064fe219a82a942-9d360f706a9d33ca-01, grpc-accept-encoding: gzip, grpc-timeout: 957031u] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false
```

I've enabled the logging exporter and see a span created, but this is from one of the methods manually annotated using `@WithSpan` and ends up being the root span. The traceparent from the gRPC inbound headers is never extracted and added to the context.
```
Logging Exporter: MyCompanyService.serviceMethod 728eab8cb77c06d54eb8559f68a1d267 1d196d9364f409ab AttributeKeyImpl{getType=STRING, key=thread.name}="grpc-default-executor-1" AttributeKeyImpl{getType=LONG, key=thread.id}=49 
```

I would really appreciate any guidance on how to debug this. I enabled debug logging on the agent and found one log message related to `TracingServerInterceptor` and a skipped instrumentation message

```
[opentelemetry.auto.trace 2020-10-19 22:45:31:258 -0700] [main] DEBUG io.opentelemetry.javaagent.tooling.Instrumenter$Default - Instrumentation skipped, mismatched references were found: [grpc] -- io.opentelemetry.javaagent.instrumentation.grpc.v1_5.GrpcServerBuilderInstrumentation on org.springframework.boot.loader.LaunchedURLClassLoader@2513a118
[opentelemetry.auto.trace 2020-10-19 22:45:49:561 -0700] [main] DEBUG io.opentelemetry.javaagent.tooling.Instrumenter$Default - -- io.opentelemetry.javaagent.shaded.instrumentation.grpc.v1_5.server.TracingServerInterceptor:69 Missing class io.opentelemetry.javaagent.shaded.io.grpc.Contexts
```

I have attached a debugger to the agent and I believe the problem is due to `muzzle.matches(classLoader)` returning false: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/Instrumenter.java#L175
It also generated the skipped grpc instrumentation log message but I am not sure how to fix the issue.

**Comment by trask** [comment]
Hi @fuleow, thanks for reporting this! @anuraaga just noticed this same issue and is fixing in #1426 👍

**Comment by iNikem** [comment]
@trask so this is closed by #1426 ?

**Comment by fuleow** [comment]
Thanks @trask I will close this issue. I've downgraded to 0.8.0 for now and it's working.
